sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -520,6 +522,20 @@ def process(
             negative_prompts=all_negative_prompts,
         )
 
+        if opts.dp_write_raw_template:
+            # if original_prompt / original_negative_prompt will not be auto quoted by webui and will be effected by stripped, we need to quote it ourselves
+            if original_prompt and original_prompt != original_prompt.strip() and original_prompt == generation_parameters_copypaste.quote(original_prompt):
+                original_prompt = json.dumps(original_prompt, ensure_ascii=False)
```

Comment:
This bit is now duplicated – maybe `requote_prompt(prompt: str) -> str | None`?

Reply:
yeah that would look cleaner